Cingulara/openrmf-api-upload
# release

- updating the version
- updating documentation
- Adding initial tests for CI/CD using GH Actions
- #42 Adding a better health check
- adding Jaeger Tracing framework
- Fixing the checklist title
- 0.12.2 add IP address from SCAP
- update to pull the first IP address on SCAP
- Adding the Nessus filename
- updating the version number
- upload good and bad checklists tracking
- reading for the checklist version
- putting the latest version tag in here
- Adding Nessus SCAP scan parsing
- Adding Jaeger Tracing
- updating tag to search on based on cdf: or xccdf:
- adding reading of Nessus SCAP scan results XML
- updated message for auditing
- #20 updating the messages on audits
- #20 adding audit information
- updated metrics for prometheus
- updating copyright notice
- adding logging for steps in the APIs
- updating API documentation
- updated copyright notice
- Naming the NATS client;
- #34 updating the system checklist count eventually
- #34 allow upload with a system info and checklists
- Update issue templates
- #32 NATS client options and reconnect
- #31 fixing the lookup of existing checklist
- #31 allow update checklist with a SCAP scan XML
- updating version number
- #28 adding the hostname from the SCAP to the CKL
- #28 allowing SCAP upload to generate new checklist
- #28 allow upload of XCCDF SCAP for generating CKL
- Fixing extra \n in files between XML tags
- put all ENV in all caps
- updating NATS IP
- fixing code for RBAC and API to API comms
- update NATS client and subjects
- finalizing the publishing for new uploads
- Fixing post to use createdBy for userId
- adding the UserID on the upload
- updating info to version 0.8
- updating to use AuthN and AuthZ
